Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebView.cs

```diff
@@ -0,0 +1,1046 @@
+        /// <seealso cref="WebViewControlSettings.IsScriptNotifyAllowed" />
+        /// <seealso cref="WebViewControlSettings.IsJavaScriptEnabled" />
+        /// <seealso cref="WebViewControlSettings.IsIndexedDBEnabled" />
+        /// <remarks>Use the <see cref="WebViewControlSettings" /> object to enable or disable the use of JavaScript, ScriptNotify, and IndexedDB in the <see cref="WebView"/>.</remarks>
+        [Browsable(false)]
+        public WebViewControlSettings Settings
+        {
+            get
+            {
+                VerifyAccess();
+                Verify.IsNotNull(_webViewControl);
+                return _webViewControl?.Settings;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the Uniform Resource Identifier (URI) source of the HTML content to display in the <see cref="WebView"/>.
+        /// </summary>
+        /// <value>The Uniform Resource Identifier (URI) source of the HTML content to display in the <see cref="WebView"/>.</value>
+        public Uri Source
+        {
+            get => (Uri)GetValue(SourceProperty);
+            set => SetValue(SourceProperty, value);
+        }
+
+        /// <summary>
+        /// Gets the version of EDGEHTML.DLL used by <see cref="WebView"/>.
+        /// </summary>
+        /// <value>The version of EDGEHTML.DLL used by <see cref="WebView"/>.</value>
+        [Browsable(false)]
+        public Version Version
+        {
+            get
+            {
+                VerifyAccess();
+                Verify.IsNotNull(_webViewControl);
+                return _webViewControl?.Version;
+            }
+        }
+
+        private bool WebViewControlInitialized => _webViewControl != null;
+
+        /// <summary>
+        /// Closes the <see cref="T:Microsoft.Toolkit.Win32.UI.Controls.WPF.WebViewHost" />.
+        /// </summary>
+        /// <seealso cref="M:System.IDisposable.Dispose" />
+        public override void Close()
+        {
+            // TODO: Guard IsDisposed
+            UnsubscribeEvents();
+            _webViewControl?.Close();
+            _webViewControl?.Dispose();
+
+            _webViewControl = null;
+            _process = null;
+        }
+
+        /// <summary>
+        /// Gets the deferred permission request with the specified <see cref="WebViewControlPermissionRequest.Id" />.
+        /// </summary>
+        /// <param name="id">The <see cref="WebViewControlPermissionRequest.Id" /> of the deferred permission request.</param>
+        /// <returns><see cref="WebViewControlDeferredPermissionRequest" /> The deferred permission request with the specified <see cref="WebViewControlPermissionRequest.Id" />, or null if no permission request with the specified <see cref="WebViewControlPermissionRequest.Id" /> was found.</returns>
+        public WebViewControlDeferredPermissionRequest GetDeferredPermissionRequestById(uint id)
```

Comment:
@normesta @mcleanbyron This is an example of where we have a different method signature than WebViewControl